src/FlightMap/Widgets/PhotoVideoControl.qml

```diff
@@ -466,6 +466,7 @@ Rectangle {
                                 minimumValue:               parent._fact.min
                                 stepSize:                   parent._fact.increment
                                 visible:                    parent._isSlider
+                                value:                      parent._fact.value
```

Comment:
I don't understand why this is needed given the `Component.onCompleted` call below?

Reply:
I had to add this to get the sliders to behave as expected. Otherwise they are reset to their minimum value every time the camera settings dialog is displayed.

Reply:
There was a Qt bug with sliders with respect to setting the value property like this. You'll see in other usage of this control it follows the pattern of setting the value in onCompleted to work around it (look at SurveItemEditor/qml for an example). I don't have a camera that can show these sliders and SITL doesn't expose them either. So I have nothing to test this with. It should work fine with the onCompleted since I was able to verify that it works correctly in the other usages of it. I don't want to change to the value property setting to expose what I remember as a subtle bug in the control. Can you debug this such to determine why the onCompleted isn't working correctly? Are you using a different version of Qt than what is specific (5.12.6)?